healthcheck/client.go

```diff
@@ -0,0 +1,61 @@
+package healthcheck
+
+import (
+	"github.com/ONSdigital/go-ns/rhttp"
+	"net/http"
+	"fmt"
+)
+
+// healthcheckClient is an implementation of Client that can be used to call the healthcheck endpoint of any service
+type healthcheckClient struct {
+	Client  httpClient
+	Url     string
+	Service string
```

Comment:
Consider making these struct members private to encapsulate them. When they are private they must be injected via the constructor function